src/commands/checkModel.ts

```diff
@@ -44,6 +45,20 @@ export async function checkModel(
     if (!uri) {
         return;
     }
+
+    // Unless the given filePath already starts with 'MC', prepend MC to the name
+    // and check if it exists. If yes, it becomes the spec file.  If not, fall back
+    // to the original file.
+    const b = Utils.basename(uri);
+    if (!b.startsWith('MC')) {
+        const n = uri.toString().replace(b.toString(), 'MC' + b);
+        const specFiles = await getSpecFiles(vscode.Uri.parse(n), false);
+        if (specFiles) {
+            doCheckModel(specFiles, true, extContext, diagnostic, true);
+            return;
+        }
+  
```

Comment:
It looks like the closing brace is missing here.

Reply:
These lines duplicate the code that comes after the new `if`. I guess, we could just have the following flow, which is easier to read and reason about:

```typescript
if (!startsWith(...)) {
   specFiles = ...
}
if (!specFiles) {
   specFiles = ... // what we have now
}
if (!specFiles) {
   return;
}
doCheckModel(...)
```